tests/integration.rs

```diff
@@ -5931,6 +5959,96 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
+        #[test]
+        fn test_windows_guest_netdev_hotplug() {
+            let mut windows = WindowsDiskConfig::new(WINDOWS_IMAGE_NAME.to_string());
+            let guest = Guest::new(&mut windows);
+
+            let tmp_dir = TempDir::new_with_prefix("/tmp/ch").unwrap();
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
+
+            let mut ovmf_path = workload_path.clone();
+            ovmf_path.push(OVMF_NAME);
+
+            let mut osdisk_path = workload_path;
+            osdisk_path.push(WINDOWS_IMAGE_NAME.to_string());
+
+            let api_socket = temp_api_path(&tmp_dir);
+
+            let mut child = GuestCommand::new(&guest)
+                .args(&["--api-socket", &api_socket])
+                .args(&["--cpus", "boot=2,kvm_hyperv=on"])
+                .args(&["--memory", "size=2G,hotplug_size=5G"])
+                .args(&["--kernel", ovmf_path.to_str().unwrap()])
+                .default_disks()
+                .args(&["--serial", "tty"])
+                .args(&["--console", "off"])
+                .args(&["--net", "tap="])
+                .capture_output()
+                .spawn()
+                .unwrap();
+
+            // Wait to make sure Windows boots up
+            thread::sleep(std::time::Duration::new(60, 0));
+
+            let r = std::panic::catch_unwind(|| {
+                let auth = windows_auth();
+
+                // Initially present network device
+                let netdev_num = 1;
+                assert_eq!(get_netdev_count_windows(&auth), netdev_num);
```

Comment:
Nit: could be cleaner with `assert_eq!(get_netdev_count_windows(&auth), 1);`

Reply:
Hmm, the value is used more than in one place, that's why it's a var i think :)  Done same way in other tests.

Thanks

Reply:
Fixed teh commit message, though.

Thanks.